applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -91,6 +92,8 @@ LOG_MODULE_REGISTER(audio_datapath, CONFIG_AUDIO_DATAPATH_LOG_LEVEL);
 /* How often to print under-run warning */
 #define UNDERRUN_LOG_INTERVAL_BLKS 5000
 
+NET_BUF_POOL_FIXED_DEFINE(pool_i2s_rx, FIFO_NUM_BLKS, BLK_STEREO_SIZE_OCTETS, 8, net_buf_destroy);
```

Comment:
This could in theory also be moved to `audio_datapath_i2s_blk_complete` if you want to reduce number of global variables :)